tensorflow_probability/python/stats/kendalls_tau.py

```diff
@@ -0,0 +1,166 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Implements Kendall's Tau metric and loss."""
+
+import warnings
+
+import numpy as np
+import tensorflow as tf
+
+
```

Comment:
nit: please add a module-level constant `__all__ = ['kendalls_tau']`, following the TFP [style guide](https://github.com/tensorflow/probability/blob/master/STYLE_GUIDE.md)

Reply:
Done.